core/root/config/templates/stack-template.json

```diff
@@ -97,6 +97,11 @@
     }{{end}}
   },
   "Outputs": {
+    "KubeAwsVersion": {
+      "Description": "This stack is created by kube-aws version",
```

Comment:
Suggestion: `The version number of kube-aws which was used to create this cluster`